specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -1629,6 +1629,46 @@
           "$ref": "#/definitions/GenericDatasetTypeProperties"
         }
       }
+    },
+    "SapOpenHubTableDataset": {
+      "x-ms-discriminator-value": "SapOpenHubTable",
+      "description": "Sap Business Warehouse Open Hub Destination Table properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Dataset"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Sap Business Warehouse Open Hub Destination Table properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SapOpenHubTableDatasetTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "SapOpenHubTableDatasetTypeProperties": {
+      "description": "Sap Business Warehouse Open Hub Destination Table properties.",
+      "properties": {
+        "openHubDestinationName": {
+          "type": "object",
```

Comment:
Is it possible to define an entity for the type here (string/expression) rather than declaring as object?

Reply:
Hi @antmarti-microsoft, thanks for your comments. it looks all the other datasets have been using the object as the type. could you please share a little bit more information on the criteria when the object will be used comparing with when the specific type(e.g. string) will be used?  I recalled that the other objects were changed from specific types to object some time ago for some reasons. Thanks.

Reply:
I move this change to https://github.com/Azure/azure-rest-api-specs/pull/5195. Thanks.